x-pack/plugins/cloud_security_posture/server/saved_objects/migrations/csp_rule_template.ts

```diff
@@ -28,7 +28,7 @@ function migrateCspRuleMetadata(
       muted,
       metadata: {
         ...metadata,
-        benchmark: { ...benchmark, id: 'cis_k8s', rule_number: '' },
+        benchmark: { ...benchmark, id: 'cis_k8s', rule_number: '', posture_type: 'kspm' },
```

Comment:
assume previous versions were `kspm` since `cspm` is introduced  in 8.7 to kibana